pilot/pkg/networking/common/common.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Istio Authors
```

Comment:
We already have `pkg/networking/util`; utility/common classes are an anti-pattern to being with, I'd prefer we keep them to a minimum. In this case, just fold this code into the `util` package rather than creating a whole new package to hold three constants.

Reply:
SGTM. Will do.